aea/protocols/oef/serialization.py

```diff
@@ -90,7 +90,8 @@ def decode(self, obj: bytes) -> Message:
             new_body["agents"] = list(json_msg["agents"])
         elif oef_type in {OEFMessage.Type.OEF_ERROR}:
             operation = json_msg["operation"]
```

Comment:
Ok I see that this is necessary because of line 64 where the operation is cast to a string. Is casting to a string in line 64 necessary? Otherwise we could just not cast to strong and your fix would not be needed.